README.md

```diff
@@ -1,2 +1,15 @@
 # code-katas
-- This repo holds solutions for the Code Wars katas which I have worked on.
+- This repo holds solutions for the Code Wars katas and other katas
+which I have worked on.
+
+
+This one features a node search for airplane travel.
+
+https://codefellows.github.io/sea-python-401d4/assignments/kata_flight_paths.html
+
+### usage  in iPython3:
+- run flight_paths.py
+- data = extract_data_from_json('cities_with_airports.json')
+- gr = build_graph(data)
+- depth_first_search(gr, gr.node_dict['Denver'], gr.node_dict['Honolulu'])
+- spt_Dijkstra(gr, 'Denver', 'Honolulu') OR spt_AStar(gr, 'Denver', 'Honolulu')
```

Comment:
While I appreciate wanting to use what you've already written and not have to rewrite things, it's worth noting that you have print statements in Dijkstra's and A-Star that make the output for each extremely verbose. I'm willing to bet that Will hasn't had a chance to get to your submission for your shortest path assignment yet, but for THIS assignment I just want the output to be the shortest path between the two points and the total distance of that path. Right now I get the shortest distance, which is nice.